.github/workflows/agent-build.yml

```diff
@@ -6,7 +6,7 @@ on: [push]
 # Agent Docker image tests
 jobs:
   docker-json:
-    uses: scalyr/scalyr-agent-2/.github/workflows/docker-and-k8s.yml@master
+    uses: scalyr/scalyr-agent-2/.github/workflows/docker-and-k8s.yml@add-images-prod-push
```

Comment:
I assume this testing branch name change will be removed before merging, right? :)

Reply:
Right, it seems that's pointless and we need to use it from master)